target/linux/brcmiproc/image/Makefile

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright (C) 2016-2017 Yousong Zhou <yszhou4tech@gmail.com>
+# Copyright (C) 2018 Jianhui Zhao <jianhuizhao329@gmail.com>
+# Copyright (C) 2019 Samik Gupta <samik.gupta@broadcom.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+include $(TOPDIR)/rules.mk
+include $(INCLUDE_DIR)/image.mk
+
+DTS_DIR := $(DTS_DIR)/broadcom
+
+define Device/ps225
+	DEVICE_VENDOR := Broadcom
+	DEVICE_MODEL := PS225-HXX
+	DEVICE_DTS_DIR := $(DTS_DIR)/stingray
+	DEVICE_DTS := bcm958802a802x
+endef
+TARGET_DEVICES += ps225
```

Comment:
This block should become:

```
ifeq ($(SUBTARGET),bcm58800)
define Device/broadcom_ps225-hxx
	DEVICE_VENDOR := Broadcom
	DEVICE_MODEL := PS225-HXX
	DEVICE_DTS_DIR := $(DTS_DIR)/stingray
	DEVICE_DTS := bcm958802a802x
endef
TARGET_DEVICES += broadcom_ps225-hxx
endif
```

I'm not sure about the best model name yet, but this should be a step forward.

Reply:
Changed the block to match the code above